srcpkgs/mgba/template

```diff
@@ -1,18 +1,21 @@
 # Template file for 'mgba'
 pkgname=mgba
-version=0.8.4
+version=0.9.2
 revision=1
 build_style=cmake
 hostmakedepends="pkg-config qt5-host-tools qt5-qmake desktop-file-utils"
-makedepends="SDL2-devel ffmpeg-devel libedit-devel libepoxy-devel libmagick-devel
- libzip-devel qt5-multimedia-devel qt5-tools-devel"
+makedepends="SDL2-devel ffmpeg-devel libedit-devel libepoxy-devel libmagick-devel libzip-devel qt5-multimedia-devel qt5-tools-devel"
```

Comment:
Please break at ~80 columns.

Reply:
agh
and i only changed it cause i thought itd cause issues ^^"
will revert that one